platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -93,12 +94,16 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(LandingPage.class);
 
+  @SuppressWarnings("squid:S2226" /* Lifecycle managed by blueprint. */)
```

Comment:
If you have a suppression on top of the class it will apply to all the methods.

Reply:
It does, but If I didn't suppress the class I would have had to put a suppression on every class variable, and that is super messy. And after taking off that suppression the only bugs were the `Squid:S2226`s. I can always change it to suppress every instance variable if you think that's better.

Reply:
Sorry I wasn't clear on what I meant. You have the same suppression on the class and on three variables. You don't need the extra suppressions you have on lines 54, 97 and 100 because of this class suppression.